<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dtumodifyadjustDlg</class>
 <widget class="QDialog" name="dtumodifyadjustDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>106</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>106</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>106</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLabel" name="text4">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>修改值</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="channel_no">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="text1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>通道号</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="text3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>修改参数</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEdit_modify">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btn_modify">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="comboBox_param">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_modify</sender>
   <signal>clicked()</signal>
   <receiver>dtumodifyadjustDlg</receiver>
   <slot>modify_adjust()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_param</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>dtumodifyadjustDlg</receiver>
   <slot>show_infomation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>modify_adjust()</slot>
  <slot>show_infomation(int)</slot>
 </slots>
</ui>
